### PR TITLE
fix : bottom sheet if fully

### PR DESCRIPTION
**Pull request by afandi-siregar**
fix : https://3.basecamp.com/5190892/buckets/31702802/card_tables/cards/6043471306

### CODE REVIEW

File: components/common/BottomSheet.vue [hunk 1]
```diff
@@ -94,18 +99,23 @@ function toggleBodyScroll(enable: boolean) {
 }
 
 function setSheetHeight(value: number) {
-  const maxHeight = 95;
+  const maxHeight = 100;
   sheetHeight =
     value < maxHeight ? Math.max(0, Math.min(100, value)) : maxHeight;
+
+  console.log("setSheetHeight", sheetHeight);
```
Comment (afif-hh) [inline components/common/BottomSheet.vue:106]: remove

File: partial/restaurant_detail/RestaurantHeader.vue [hunk 2]
```diff
@@ -244,6 +260,14 @@ const selectedReviewsData = computed(() => {
   )[0];
 });
 
+const titleBottomSheet = computed(() => {
```
Comment (afif-hh) [inline partial/restaurant_detail/RestaurantHeader.vue:263]: diganti ke bottomSheet title,

File: components/common/BottomSheet.vue [hunk 3]
```diff
@@ -94,18 +99,23 @@ function toggleBodyScroll(enable: boolean) {
 }
 
 function setSheetHeight(value: number) {
-  const maxHeight = 95;
+  const maxHeight = 100;
```
Comment (afif-hh) [inline components/common/BottomSheet.vue:102]: maxHeight nya di jadiin props aja, default nya 95, jgn di setting semua jadi 100

**Review by afif-hh — CHANGES_REQUESTED** [review]

**Comment by afandi-siregar** [comment]
done mas @afif-hh

**Review by afif-hh — APPROVED** [review]

**Pull request merged**